Database/Corrections/mopQuestFixes.lua

```diff
@@ -1615,6 +1628,9 @@ function MopQuestFixes.Load()
         [30322] = { -- Money Matters
             [questKeys.preQuestSingle] = {30257},
         },
+        --[30325] = { -- Where it Counts
+            --[questKeys.objectives] = {{{59123}},nil,{{210890,210955}}},
+       -- },
```

Comment:
```suggestion
        [30325] = { -- Where it Counts
            [questKeys.objectives] = {{{59123}},nil,{{210890},{210955}}},
        },
```

This quest also needs to be added with `QuestieCorrections.objectObjectiveFirst[30325] = true` to the top of the file, so that the order matches.